.github/workflows/build.yml

```
@@ -31,7 +31,14 @@ permissions:
   contents: read
 
 jobs:
+  should-run:
+    if: github.event_name != 'push' || !contains(github.event.head_commit.message, ':bookmark: bump version')
```

actionlint (1.7.10)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)